tasks/marin_l_cnt_mismat_chrt_in_two_str/mpi/src/ops_mpi.cpp

```diff
@@ -0,0 +1,63 @@
+#include "marin_l_cnt_mismat_chrt_in_two_str/mpi/include/ops_mpi.hpp"
+
+#include <mpi.h>
+
+#include <algorithm>
+#include <cstddef>
+#include <string>
+
+#include "marin_l_cnt_mismat_chrt_in_two_str/common/include/common.hpp"
+
+namespace marin_l_cnt_mismat_chrt_in_two_str {
+
+MarinLCntMismatChrtInTwoStrMPI::MarinLCntMismatChrtInTwoStrMPI(const InType &in) {
+  SetTypeOfTask(GetStaticTypeOfTask());
+  GetInput() = in;
+  GetOutput() = 0;
+}
+
+bool MarinLCntMismatChrtInTwoStrMPI::ValidationImpl() {
+  return true;
+}
+
+bool MarinLCntMismatChrtInTwoStrMPI::PreProcessingImpl() {
+  GetOutput() = 0;
+  return true;
+}
+
+bool MarinLCntMismatChrtInTwoStrMPI::RunImpl() {
```

Comment:
Please, ensure the data is properly split and scattered from rank 0 to other ranks